tensorflow/java/README.md

```diff
@@ -120,8 +120,8 @@ libraries will need to be built from source.
       //tensorflow/java:libtensorflow_jni
     ```
 
-The JAR (`libtensorflow.jar`) and native library (`libtensorflow_jni.so`) will 
-be in `bazel-bin/tensorflow/java`.
+The JAR (`libtensorflow.jar`) and native library (`libtensorflow_jni.dylib`) will 
```

Comment:
Hey @asimshankar! Bazel creates the `dylib` file as well which is needed in order to run any of the logic in the tensorflow jar that calls out to the native code (see https://github.com/tensorflow/tensorflow/tree/master/tensorflow/java#quickstart step 4). I am not sure what the `libtensorflow.so` file does. If you could find out on your end that would be helpful! I don't think I am using it though (running on my mac).